modules/client/src/interfaces.ts

```diff
@@ -281,16 +298,18 @@ type Erc20TokenTransfer = BaseTokenTransfer & {
   };
 };
 
-export type Deposit = (NativeTokenTransfer | Erc20TokenTransfer) & {
-  from: string;
-  type: TransferType.DEPOSIT;
-};
+export type Deposit =
+  & (NativeTokenTransfer | Erc20TokenTransfer | Erc721TokenTransfer)
```

Comment:
Is it OK to start with `&` after no particle?

Reply:
This is how the formatter left it, looked weird to me too